fiftyone/plugins/context.py

```diff
@@ -136,6 +136,14 @@ def register_all(self):
             sys.modules[module.__name__] = module
             spec.loader.exec_module(module)
             module.register(self)
+        except ModuleNotFoundError as e:
```

Comment:
I don't think we should disable the plugin here, for a few reasons:
- Maybe other components of the plugin will still function without the Python operators
- We don't disable the plugin a few lines below this if things fail for other reasons. Prefer consistency

FWIW I'm not a fan of automatically disabling the users stuff in either case. Import failures are already non-fatal (we log a warning), so better to raise the error every time so the user either manually disables the plugin or fixes the issue.

Reply:
I disagree. If a module is missing, the plugin shouldn't work at all/will not work and it's failure should not keep spamming the logs. Also for the delegated operators case, it'll keep trying to run an operator that cannot be imported.